### ISSUE TITLE
Stored properties cannot be marked unavailable with '@available' 

### ISSUE THREAD

**Issue by AizatQisti** [BeauNouvelle/SimpleCheckbox#39]
have issue running on xcode 15 error "Stored properties cannot be marked unavailable with '@available' "

**Comment by BeauNouvelle** [comment]
Ah thanks for this. Will check it out for you!

**Comment by Abduladzhi** [comment]
<img width="1116" alt="Screenshot 2023-10-10 at 01 59 39" src="https://github.com/BeauNouvelle/SimpleCheckbox/assets/99020333/d2c2e9df-3e7a-49d2-8d61-405147fa05f5">

**Comment by triggs** [comment]
I have the same issue, i've checked the latest code and that attribute is not there.
(i don't really understand how cocoapods works under hood but just in case this helps)

on the main page of the repo it says the latest release is 2.5.0 but when i check the podfile.lock it says its 2.1.0 
maybe there is a config issue with pod?

**Comment by BeauNouvelle** [comment]
Sorry about this. Looks like the pod file hasn't been updated. Will get onto it

**Comment by QaziAmmar** [comment]
facing the same issue updating the pod file as well.

**Comment by BeauNouvelle** [comment]
@QaziAmmar 

Could you try again with 2.5.1?

It looks like I made the change some months ago, but pods didn't pick it up until I created a new tag.

**Comment by vit-azarov** [comment]
@BeauNouvelle 
Hello! I just upgraded my xcode to 15.3 and also got the above error.
With `pod 'SimpleCheckbox'` in my pod file, 2.1.0 is being installed on `pod update`
If I specify a version with `pod 'SimpleCheckbox', '~> 2.5.1'` (as you mentioned above) i got this error on pod update:

> [!] CocoaPods could not find compatible versions for pod "SimpleCheckbox":
>   In Podfile:
>     SimpleCheckbox (~> 2.5.1)
> 
> None of your spec sources contain a spec satisfying the dependency: `SimpleCheckbox (~> 2.5.1)`.
> 
> You have either:
>  * mistyped the name or version.
>  * not added the source repo that hosts the Podspec to your Podfile.

Can you please suggest what can be a solution here?

**Comment by BeauNouvelle** [comment]
Ah perhaps I need to push the update to the cocoa pods repo. Sorry, I've been off cocoa pods for years now. Is there any reason why you don't want to use SPM?

Not sure I have the time to maintain a cocoa pods supported version of this for what is essentially a few small files.

**Comment by vit-azarov** [comment]
No real reason, just a fact that we use Cocoapods for all dependencies in our project. I migrated to SPM today without any issues, since you are not supporting pods. All works good, thank you!

**Comment by Neoron95** [comment]
I also have issue because cocoapods.org has only 2.1 version

You can simply use this, for downloading directly latest version from this repo
`pod 'SimpleCheckbox', :git => 'https://github.com/BeauNouvelle/SimpleCheckbox.git'`

**Comment by sultan-arshi** [comment]
+1

**Comment by BeauNouvelle** [comment]
Yeah sorry everyone. I stopped supporting cocoa pods when cocoa pods stopped active development.
